newIDE/app/src/UI/Theme/SolarizedDarkTheme/index.js

```diff
@@ -0,0 +1,231 @@
+// @flow
+import { type Theme } from '../DefaultTheme';
+import { createMuiTheme, darken, lighten } from '@material-ui/core/styles';
+import './Mosaic.css';
+import './EventsSheet.css';
+import 'react-virtualized/styles.css'; // Styles for react-virtualized Table
+import './Table.css';
+import './Markdown.css';
+
+const almostWhite = '#EEE';
+const lightWhite = '#DDD';
+const notSoWhite = '#CCC';
+const solarizedDark1 = '#047AA6';
+const solarizedDark2 = '#047AA6';
+
+// Use the fonts provided by the operating system(s) as possible.
+// If you update this font list, be sure to do it in all the other places using fonts in the codebase.
+const fontFamily =
+  "-apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen-Sans, Ubuntu, 'Helvetica Neue', Helvetica, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol'";
+
+const systemSelectionColor = '#4c92ff'; //OS X selection
+
+/**
+ * The background color of the main window
+ */
+const backgroundColor = '#00212B';
+
+/**
+ * The background color of the "papers", "dialogs", etc...
+ */
+const canvasColor = '#002B36';
+
+/**
+ * The alternate background color, for some lists or search box,
+ * to distinguish them from other content.
+ */
+const alternateCanvasColor = '#003847';
+
+// GDevelop specific variables:
+const gdevelopTheme = {
+  palette: {
+    canvasColor,
+  },
+  message: {
+    warning: '#ffa500',
+    error: '#f00',
+    valid: '#00db00',
+  },
+  toolbar: {
+    backgroundColor: backgroundColor,
+    separatorColor: '#047AA6',
+  },
+  closableTabs: {
+    fontFamily,
+    containerBackgroundColor: backgroundColor,
+    backgroundColor: canvasColor,
+    textColor: '#878787',
+    selectedBackgroundColor: solarizedDark1,
+    selectedTextColor: almostWhite,
+    width: 400,
+    height: 32,
+    closeButtonWidth: 24,
+  },
+  imageThumbnail: {
+    selectedBorderColor: systemSelectionColor,
+  },
+  list: {
+    itemsBackgroundColor: alternateCanvasColor,
+  },
+  searchBar: {
+    backgroundColor: alternateCanvasColor,
+  },
+  listItem: {
+    groupBackgroundColor: backgroundColor,
+    groupTextColor: '#AAA',
+    deprecatedGroupTextColor: '#888',
+    separatorColor: alternateCanvasColor,
+    selectedBackgroundColor: systemSelectionColor,
+    selectedTextColor: almostWhite,
+    rightIconColor: '#fff',
+    selectedRightIconColor: '#fff',
+    errorTextColor: '#ff2e16',
+    warningTextColor: '#ffb032',
+    selectedErrorBackgroundColor: '#ff2e16',
+    selectedWarningBackgroundColor: '#ffb032',
+  },
+  emptyMessage: {
+    shadowColor: '#000',
+  },
+  logo: {
+    src: 'res/GD-logo-big.png',
+  },
+  mosaicRootClassName: 'mosaic-gd-dark-theme', // See Mosaic.css
+  eventsSheetRootClassName: 'gd-events-sheet-dark-theme', // See EventsSheet.css
```

Comment:
Here replace every dark- or nord- by solarized-dark- so that your CSS files rules are used

Reply:
Thank you. Previous I used xxx-solarised-dark-theme but when I did that the css stopped working and I had to spend hours going through the codes in the ui. I never knew it was right in the index folder..